src/pages/RoomInvitePage.js

```diff
@@ -93,13 +93,22 @@ function RoomInvitePage(props) {
         const sections = [];
         let indexOffset = 0;
 
+        // Filter all options that is a part of the search term or in the personal details
```

Comment:
This filter should only happen when we have searchTerm.
For reference: https://github.com/Expensify/App/blob/main/src/libs/OptionsListUtils.js#L1697

Reply:
The point you made is correct. But what I am suggesting is an optimization. Like when there is no searchTerm, we will always get the whole list. So it becomes a wasteful search(Correct me if wrong).
 So we can filter only when we have a searchTerm,
 like the below pseudo code
```
const filterSelectedOptions = selectedOptions
if searchTerm != '' {
   filterSelectedOptions = filter()
}
```

@dukenv0307